apple/testing/default_runner/macos_test_runner.bzl

```diff
@@ -14,24 +14,27 @@
 
 """Sample Apple test runner rule."""
 
-load(
-    "@build_bazel_apple_support//lib:xcode_support.bzl",
-    "xcode_support",
-)
 load(
     "@build_bazel_rules_apple//apple/testing:apple_test_rules.bzl",
     "AppleTestRunnerInfo",
 )
 
 def _get_xctestrun_template_substitutions(ctx):
     """Returns the template substitutions for the xctestrun template."""
-    xcode_version = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
+    xcode_version = ctx.attr._xcode_config[apple_common.XcodeVersionConfig].xcode_version()
+
+    # XCode version is not present on non-macOS platforms
+    if not xcode_version:
+        return {
+            "%(xctestrun_insert_libraries)s": "",
+        }
 
     # Xcode 10 introduced new dylibs that are required when running unit tests, so we need to
     # provide different values in the xctestrun file that describes the test.
     # TODO(kaipi): Revisit this when Nitro has support for macOS. Nitro should be the one detecting
     #              Xcode version and configuring it appropriately.
-    if xcode_support.is_xcode_at_least_version(xcode_version, "10.0"):
```

Comment:
Since it's already returning above, I think it's safe to use `is_xcode_at_least_version` here?